tests/integration/__init__.py

```diff
@@ -240,6 +250,19 @@ def teardown_package():
             log.warning('Did not find cluster: %s' % cluster_name)
 
 
+def execute_until_pass(session, query):
+    while True:
+        try:
+            return session.execute(query)
+        except ConfigurationException:
+            # keyspace/table was already created/dropped
+            return
+        except (OperationTimedOut, ReadTimeout, ReadFailure, WriteTimeout, WriteFailure):
+            ex_type, ex, tb = sys.exc_info()
+            log.warn("{0}: {1} Backtrace: {2}".format(ex_type.__name__, ex, traceback.extract_tb(tb)))
+            del tb
+
+
```

Comment:
In this case, we do want to try indefinitely until the query passes. As we handle the expected exceptions, anything else will be raised to the caller. We could have a max_tries = 100 for example, but we need to do some testing to make sure it can handle the worst case scenario in the current test suite.